scrapyd_k8s/api.py

```diff
@@ -104,7 +106,21 @@ def api_listjobs():
 def error(msg, status=200):
     return { 'status': 'error', 'message': msg }, status
 
+def enable_authentication(config_username: str, config_password: str) -> BasicAuth:
```

Comment:
no type annotations are used elsewhere in this project, so let's not do that here to keep it homogeneous